src/WebJobs.Script.WebHost/Management/MeshInitServiceClient.cs

```diff
@@ -42,21 +44,58 @@ public async Task MountCifs(string connectionString, string contentShare, string
         public Task MountBlob(string connectionString, string contentShare, string targetPath)
         {
             // todo: Implement once mesh init server supports mounting blobs
-            throw new NotImplementedException();
+            throw new NotImplementedException(nameof(MountBlob));
         }
 
         public async Task MountFuse(string type, string filePath, string scriptPath)
-            => await Mount(new[]
+            => await SendAsync(new[]
             {
                 new KeyValuePair<string, string>("operation", type),
                 new KeyValuePair<string, string>("filePath", filePath),
                 new KeyValuePair<string, string>("targetPath", scriptPath),
             });
 
-        private async Task Mount(IEnumerable<KeyValuePair<string, string>> formData)
+        public async Task PublishContainerFunctionExecutionActivities(IEnumerable<ContainerFunctionExecutionActivity> activities)
         {
-            var res = await _client.PostAsync(_environment.GetEnvironmentVariable(EnvironmentSettingNames.MeshInitURI), new FormUrlEncodedContent(formData));
+            var meshInitServiceUri = _environment.GetEnvironmentVariable(EnvironmentSettingNames.MeshInitURI);
+            _logger.LogInformation($"Publishing {activities.Count()} function execution activities to {meshInitServiceUri}");
+
+            foreach (var activity in activities)
+            {
+                try
+                {
+                    await Utility.InvokeWithRetriesAsync(async () =>
+                    {
+                        await PublishContainerFunctionExecutionActivity(activity);
```

Comment:
Why are you sending these one at a time? I thought the whole point with the batching was to reduce the requests?